envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadataValidator.kt

```diff
@@ -57,6 +58,17 @@ class NodeMetadataValidator(
         request?.node?.let { validateV3Metadata(it) }
     }
 
+    override fun onV2StreamDeltaRequest(streamId: Long, request: DeltaDiscoveryRequest?) {
+        request?.node?.let { validateV2Metadata(it) }
+    }
+
+    override fun onV3StreamDeltaRequest(
+        streamId: Long,
+        request: io.envoyproxy.envoy.service.discovery.v3.DeltaDiscoveryRequest?
```

Comment:
Let's make alias for import `import io.envoyproxy.envoy.service.discovery.v3.DeltaDiscoveryRequest? as DeltaDiscoveryRequestV3`

Reply:
I will change all imports to *DiscoveryR*V2/3.